core/trino-main/src/test/java/io/trino/block/AbstractTestBlock.java

```diff
@@ -25,6 +25,7 @@
 import io.trino.spi.block.DictionaryBlock;
```

Comment:
```
Fix AbstractTestBlock.assertRetainedSize to take into
account added singleRowBlockWriter field to
Let's reuse SingleRowBlockWriter instance instead
of creating on every beginBlockEntry() call.
```

In description I would just focus on reusing `SingleRowBlockWriter` (e.g. why we do it)

Also attach bendhmark results within commit description

Reply:
ok, done